src/main/java/com/faforever/client/remote/domain/GameLaunchMessage.java

```diff
@@ -1,50 +1,33 @@
 package com.faforever.client.remote.domain;
 
+import com.faforever.client.fa.relay.LobbyMode;
+import com.faforever.client.game.Faction;
+import lombok.Getter;
+import lombok.NonNull;
+import lombok.Setter;
+
 import java.util.List;
 
+@Getter
+@Setter
 public class GameLaunchMessage extends FafServerMessage {
 
+  /**
+   * Stores game launch arguments, like "/ratingcolor d8d8d8d8", "/numgames 236".
+   */
   private List<String> args;
   private int uid;
   private String mod;
   private String mapname;
+  @NonNull
+  private String name;
+  private int expectedPlayers;
+  private int team;
+  private int mapPosition;
```

Comment:
What does this actually get used for?

Reply:
I think it's the slot you spawn in.

Reply:
Yea but the PR doesn’t reference it anywhere

Reply:
Indeed. I just copied the stuff from the old GW branch and did not cross-check.
In ladder the map positions somehow work out of the box.
But if you have more than 2 players and you need sorting (left vs right or top vs bottom) you will somehow need to solve that.